_data/contributors.yml

```diff
@@ -5982,10 +5984,9 @@
   github_image_id: 2364173
   title:
   sort:
-  bio: Software engineer and mathematician. I was awarded a PhD in Computer Science
-      (in the machine learning field) at Universidad Autónoma de Madrid.
+  bio: Software engineer and mathematician
   organization:
-  date_added: '2024-03-09'
+  date_added: '2024-11-22'
```

Comment:
This should never be updated as it's the date a contributor was added to this file.

Reply:
Ooops looks like I added conflicting info back in https://github.com/pyOpenSci/pyopensci.github.io/pull/530 . I didn't realize this file exists in multiple places. Did I change it in the wrong repo?

Reply:
@pllim you did everything right. I made a "test" copy of the workflow over here so we can test things every time we make changes to pyosmeta. So please consider this pr a end-to-end test in the pyosmeta not the final website file!!

The goal is to avoid troubleshooting pyosmeta in the website repo and rather to catch things PRIOR to a new release. You corrected things in the right spot!! This date update specifically is i think a bug in our workflow. i don't recall you changing the dates anywhere!!   
Does that make sense?

btw - Name was actually in our issue template so i cleaned that up too! i'm finding lots of small bugs thanks to you.